README.md

```diff
@@ -54,6 +54,13 @@ pip install retworkx
 will build retworkx for your local system from the source package and install
 it just as it would if there was a prebuilt binary available.
 
+Note: To build from source you will need to ensure you have a newer version
+of pip installed that supports PEP-517, or that you have manually installed
+`setuptools-rust` prior to running `pip install retworkx`. If you recieve an error
+about `setuptools-rust` not being found you should upgrade pip with
+`pip install -U pip` or manually install `setuptools-rust` with
+`pip install setuptools-rust` and try again.
```

Comment:
I think we could try to be more specific here e.g. `you need pip 19.1 or later to build the wheel from source` (note: I came up with an arbitrary version)

Reply:
Your guess is not far off, pep 517 support was added in pip 19.0: https://pip.pypa.io/en/stable/news/#v19-0 I'll update the text here to reflect that

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/533/commits/34c0632c9e0e004d3a97d0f3b03dc622bc0ad713